tasks/main.yml

```diff
@@ -477,6 +477,26 @@
   when:
     - mssql_tls_enable is not none
   block:
+    - block:
```

Comment:
I was putting names for blocks everywhere, I thought it is a requirement, I think it's fine without a name too though as tasks will be explaining what's going on anyway. I am afraid that newer versions of ansible-lint might complain about this too.